### DISCUSSION TITLE
Is there still an RDF zip file that has ttl files split by species?

### DISCUSSION THREAD

**Discussion by johnbraisted** [wikipathways/wikipathways-help#110] [Q&A]
Hello, Thanks for the wonderful resource. The rdf zip files used to have a structure like ./wp/<species_name>/ then separate ttl files in each species directory. Now the rdf zip file unzips to format ./authors/ and it has about 1947 ttl files, perhaps containing most species. Is the old structure still available, split by species?  

An older zip split by species had about 2300 ttl files. The current zip file has about 1947. Are all species still represented in this zip file?

Apologies if this general information is available elsewhere. Thank you. John Braisted NIH/NCATS

**Comment by khanspers** [comment]
Hi @johnbraisted, there was an issue with the rdf generation, which affected the data release for the past two months. I believe the issue has been fixed and the upcoming March 10 release should have the same structure as previous releases.

**Reply by johnbraisted** [reply]
Hi Kristina, Thanks for the prompt reply and great resource. I'll look forward to the next release, probably next week. Thanks.

**Comment by johnbraisted** [comment]
Closing issue. Thank you again for the quick reply.
